web_app/testSite.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Test</title>
+    <link rel="stylesheet" type="text/css" href="src/form.css">
+</head>
+<body>
+    <div id="task-modal" class="modal">
+        <div class="modal-content">
+        <span class="close">&times;</span>
+        <form id="frame">
```

Comment:
What if we change `<div id="new-task-form-body">` to the form and just not have anything here?? I think the placement of the form here (i.e. how elements are contained within it) may be messing with the flex